### ISSUE TITLE
Web - LHN - GBR and Amount is shown in LHN for expense deleted in OD

### ISSUE THREAD

**Issue by IuliiaHerets** [Expensify/App#50936]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v9.0.49-1
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Email or phone of affected tester (no customers):** gibethlehem@gmail.com
**Issue reported by:** Applause Internal Team

## Action Performed:
Precondition
WS set to Submit Manually
Employee is a member of a WS

As Employee
1. in OD add expense to in the WS, but don't submit it.
2. Go to ND > WS chat. notice that added expense is shown in the LHN with GBR showing owned amount.
3. Go to OD and delete the expense
4. Go to ND. and notice GBR and owned amount are shown in the LHN

## Expected Result:
GBR and owned amount are not shown in the LHN when the expense is deleted in OD

## Actual Result:
GBR and owned amount are shown in the LHN after expense is deleted in OD

## Workaround:
Unknown

## Platforms:

- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/1c0395b4-d449-495a-88fb-a903988662f6
 
</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by slafortune** [comment]
@IuliiaHerets do you think this should be closed?

**Comment by slafortune** [comment]
I don't think this is an issue as it's not actually related to deleting an expense but rather deleting a report.
